src/importlinter/contracts/layers.py

```diff
@@ -29,9 +29,21 @@ class Layer:
 
 class LevelField(fields.Field):
     def parse(self, raw_data: str | list) -> Level:
-        layers = set()
         raw_string = fields.StringField().parse(raw_data)
-        raw_items = [item.strip() for item in raw_string.split("|")]
+
+        if "|" in raw_string and ":" in raw_string:
+            raise ValueError
```

Comment:
Not sure exactly what error we should raise here, any suggestions?

Reply:
It should raise `importlinter.domain.fields.ValidationError`.

Reply:
Thanks, changed it